Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Rebuild the source code only when needed
```

Comment:
While, the standalone image isn't being used right now, it might be wise to set up a GitHub Action to build this Dockerfile in PR builds.

Reply:
Ill look into this later.